content/_releases/22/34.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Product Update - v22.34
+description: Details of the product update v22.34 from 25th of August 2022.
+layout: rel
+version: "22.34"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-08-25
+redirect_from:
+  - /releases/22.34.html
+---
+
+## Improvements and Updates
+
+### Meaningful error for invalid SSH key entry
+
+When we introduced [support for newest SSH versions](/releases/22/14#ssh-keys-and-ssh-88)
+for the user SSH keys used to deploy components, we deprecated the old `ssh-dss`
+types. Now the user-interface shows a distinct error explaining the situation if
```

Comment:
If Users enter an unsupported or incorrect SSH the user interface now displays an informative error message instead of simply disabling the 'Save' button.

Reply:
Done